Cesium3DTilesSelection/src/Tile.cpp

```diff
@@ -424,7 +426,8 @@ void Tile::releaseReference([[maybe_unused]] const char* reason) noexcept {
   // content will be unloaded immediately.
   if (!isContentReferenced(*this) && this->_pLoader &&
       this->_pLoader->getOwner()) {
-    this->_pLoader->getOwner()->markTileEligibleForContentUnloading(*this);
+    this->_pLoader->getOwner()->markTileEligibleForContentUnloading(
```

Comment:
Looking at `markTileEligible...`, indeed the `Tile` can be unloaded by this function so we need that `const_cast`. But then the `const` in `releaseReference` is basically a lie.

Is it hard to work with this if we remove the `const`? Maybe there's another place where it's more reasonable to do a `const_cast`?